### ISSUE TITLE
Add database setup instructions

### ISSUE THREAD

**Issue by jmcarp** [opencivicdata/docs.opencivicdata.org#75]
The docs should include or link to instructions for running the database locally--something like

* Install postgres
* Install postgis
* Create pupa database user
* Create opencivicdata db
* Run dbinit

Adding docker instructions would be neat too!

**Comment by jpmckinney** [comment]
I have very minimal instructions in another project which could be expanded into these docs: https://github.com/opencivicdata/scrapers-ca/#usage

**Comment by fgregg** [comment]
yeah, I have basically the same https://github.com/opencivicdata/scrapers-us-municipal#initialization

**Comment by jpmckinney** [comment]
@jmcarp On which page of the docs do you think it would make sense to add this content?

**Comment by jmcarp** [comment]
How about as a sibling to http://docs.opencivicdata.org/en/latest/scrape/run_scraper.html ? Then, instead of the oblique reference to the database ( `if don’t have an opencivicdata postgres database set up` ), that page can say something like:

`To store scraped data in a database, run (link to database setup). Otherwise, if don’t have an opencivicdata postgres database set up, ...`
